tests/phpunit/tests/media.php

```diff
@@ -3462,6 +3452,184 @@ public function data_wp_get_attachment_image_decoding_attr() {
 		);
 	}
 
+	/**
+	 * Test whether wp_decoding_enabled() returns the expected value.
+	 *
+	 * @ticket       58892
+	 * @dataProvider data_wp_decoding_enabled_tag_name_defaults
+	 *
+	 * @covers ::wp_decoding_enabled
+	 *
+	 * @param string $tag_name Tag name.
+	 * @param bool   $expected Expected return value.
+	 */
+	public function test_wp_decoding_enabled( $tag_name, $expected ) {
+		if ( $expected ) {
+			$this->assertTrue( wp_decoding_enabled( $tag_name, 'the_content' ) );
+		} else {
+			$this->assertFalse( wp_decoding_enabled( $tag_name, 'the_content' ) );
+		}
```

Comment:
```suggestion
		$this->assertSame( $expected, wp_decoding_enabled( $tag_name, 'the_content' ) );
```

Use `assertSame` instead of conditional logic

Reply:
🤦